src/validation.cpp

```diff
@@ -2399,8 +2405,8 @@ bool CChainState::ConnectBlock(const CBlock& block, BlockValidationState& state,
                 } else {
                     continue;
                 }
-
             }
+            nTime2_index += GetTimeMicros() - nTime2_index2;
```

Comment:
can we use chrono time types instead of raw values? I guess we could wait until the rest of this code is refactored to that

Reply:
out of scope PR?